ledger/eval_simple_test.go

```diff
@@ -87,14 +87,14 @@ func TestBlockEvaluator(t *testing.T) {
 	st.Sig[2] ^= 8
 	txgroup := []transactions.SignedTxn{stbad}
 	err = eval.TestTransactionGroup(txgroup)
-	require.Error(t, err)
+	require.ErrorContains(t, err, `transaction already in ledger`)
```

Comment:
Definitely not the intent.  This is a bad test that is trying to test signature problems with `TestTransactionGroup`, which doesn't check signatures.

(The test above is also misleading, because the signature is not checked)